components/NumberInput.js

```diff
@@ -13,11 +13,22 @@ class NumberInput extends Component {
     this.setState(Object.assign({}, this.state, {value: parseInt(ev.target.value)}))
   }
 
-  blur(ev) {
+  _finishEditing() {
```

Comment:
Sure thing.  Any specific style standards I should read up on?  I'm mostly flying by the seat of my pants when it comes to JS (even at the office) and pulling style as I come across it in the broad interwebs.  My intention was to make this a "private" method since it's not exposed, but then I guess the rest of the methods aren't either.